src/test/rustdoc/intra-doc/associated-items.rs

```diff
@@ -57,4 +57,12 @@ impl T2 for S {
     fn ambiguous_method() {}
 }
 
+// @has associated_items/enum.MyEnum.html '//a/@href' 'enum.MyEnum.html#variant.MyVariant'
+/// Link to [MyEnumAlias::MyVariant]
```

Comment:
This link is a valid Rust path right? Also I assume this is something that didn't work in intra-doc links before?

Reply:
Yes, https://github.com/rust-lang/rust/pull/93805#discussion_r805119510.